### PR TITLE
Feature/open source template

### PR DESCRIPTION
**Pull request by ZvonimirMedak**
## Summary

I have updated the repository to be closer in line with the new open-source-template

## Changes

- Readme.md
- Contributing.md
- Add Codeowners file
- remove bug, and feature request md files
- add bug, and feature request yml files
- update the pull_request_template.md

### Type

- [ ] **Feature**: This pull request introduces a new feature.
- [ ] **Bug fix**: This pull request fixes a bug.
- [ ] **Refactor**: This pull request refactors existing code.
- [X] **Documentation**: This pull request updates documentation.
- [X] **Other**: This pull request makes other changes.

#### Additional information

- [] This pull request introduces a **breaking change**.

### Description

- As codeowners I have added Nikola, and me
- I have updated the readme with a bit of word styling, added some new parts to it, and some code snippets
- other files were updated based on their new look in the open source template

## Checklist

- [X] I have performed a self-review of my own code.
- [X] I have tested my changes, including edge cases.
- [X] I have added necessary tests for the changes introduced (if applicable).
- [X] I have updated the documentation to reflect my changes (if applicable).

## Additional notes

The rest of the updates will be done through other pull requests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +30,7 @@ This library supports both **Swift** and **Objective-C**.
 
 ## Requirements
 
-* iOS 10 and above
+* iOS 11 and above
 * Xcode 10 and above
```
Comment (nikolamajcen) [inline README.md:34]: I'm not sure if this would work on Xcode 10.

File: README.md [hunk 2]
```diff
@@ -37,85 +39,115 @@ This library supports both **Swift** and **Objective-C**.
...
-)
-
-Sentinel.shared.setup(with: configuration)
+    let configuration = Sentinel.Configuration(
+        trigger: Triggers.shake,
+        tools: [
+            UserDefaultsTool(),
+            CustomLocationTool()
+        ],
+        preferences: optionSwitchItems
```
Comment (nikolamajcen) [inline README.md:80]: Please add `optionSwitchItems` somewhere because it is hard to understand what they are in this example.

File: README.md [hunk 3]
```diff
@@ -37,85 +39,115 @@ This library supports both **Swift** and **Objective-C**.
...
 
-The last, but not the least, is the `preferences` object which is an array of `OptionSwitchItem` objects. `OptionSwitchItem` is used to allow the user to switch of some of the preferences which are contained in the app. e.g. The app supports Analitycs and you can add an `OptionSwitchTool` which will be shown on the `Preferences` screen and the user can turn it off if he doesn't want it.
+Last, but not the least, is the `preferences` object which is an array of `OptionSwitchItem` objects. `OptionSwitchItem` is used to allow the user to switch of some of the preferences which are contained in the `UserDefaults`. To allow the user the interaction, you will have to add a `getter`, and a `setter` for the property from the `UserDefaults` object.
+
+e.g. The app supports Analitycs and you can add an `OptionSwitchTool` which will be shown on the `Preferences` screen and the user can turn it off if he doesn't want it.
+
+```swift
+    OptionSwitchItem(
```
Comment (nikolamajcen) [inline README.md:116]: Remove indentation for this example.

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1 @@
+* @nikolamajcen @ZvonimirMedak
```
Comment (nikolamajcen) [inline .github/CODEOWNERS:1]: Honestly, I'm totally fine that you set only yourself here as a maintainer. I could be added manually any time you need me 👍

**Review by nikolamajcen — APPROVED** [review]
I've added some comments, but overall, looks good 👍

File: README.md [hunk 5]
```diff
@@ -37,85 +39,115 @@ This library supports both **Swift** and **Objective-C**.
...
-            setter: { AppSwitches.analyticsEnabled = $0 },
-            getter: { AppSwitches.analyticsEnabled },
-            userDefaults: .standard,
-            userDefaultsKey: "com.infinum.sentinel.optionSwitch.analytics"
-        )
-    ]
-)
-
-Sentinel.shared.setup(with: configuration)
+    let configuration = Sentinel.Configuration(
```
Comment (nikolamajcen) [inline README.md:74]: Remove indentation.

File: README.md [hunk 6]
```diff
@@ -6,14 +6,16 @@
 
 [![Build Status](https://app.bitrise.io/app/56dc4082e9c3bb9e/status.svg?token=aHG6rIR2XDrJ3xNOIO2hXw&branch=master)](https://app.bitrise.io/app/56dc4082e9c3bb9e)
 [![Version](https://img.shields.io/cocoapods/v/Sentinel.svg?style=flat)](https://cocoapods.org/pods/Sentinel)
-[![License](https://img.shields.io/cocoapods/l/Sentinel.svg?style=flat)](https://cocoapods.org/pods/Sentinel)
+[![License](https://img.shields.io/cocoapods/l/Sentinel.svg?style=flat)](https://cocoapods.org/pods/Sentinel#license)
 [![Platform](https://img.shields.io/cocoapods/p/Sentinel.svg?style=flat)](https://cocoapods.org/pods/Sentinel)
```
Comment (nikolamajcen) [inline README.md:10]: For the better appearance, you can center this if you want :)

File: README.md [hunk 7]
```diff
@@ -37,85 +39,115 @@ This library supports both **Swift** and **Objective-C**.
...
+*NOTE: All of the subspecs add `Core` as a dependency*
+
 #### Swift Package Manager
 
 If you are using SPM for your dependency manager, add this to the dependencies in your `Package.swift` file:
 
 ```swift
 dependencies: [
-    .package(url: "https://github.com/infinum/ios-sentinel.git")
+.package(url: "https://github.com/infinum/ios-sentinel.git")
```
Comment (nikolamajcen) [inline README.md:65]: Indentation needed.
